pkg/migrations/op_add_column.go

```diff
@@ -247,7 +276,7 @@ func (o *OpAddColumn) Validate(ctx context.Context, s *schema.Schema) error {
 	return nil
 }
 
-func addColumn(ctx context.Context, conn db.DB, o OpAddColumn, t *schema.Table, tr SQLTransformer) error {
+func addColumn(ctx context.Context, conn db.DB, o OpAddColumn, t *schema.Table, fastPathDefault bool, tr SQLTransformer) error {
```

Comment:
More of a general note that we should address in a later PR is that we could make `ColumnSQLWriter` a bit smarter and handle `fastPathDefault` automatically and all the other intricate column adding methods.